src/pages/Regions/RegionEdit.tsx

```diff
@@ -99,32 +100,51 @@ const RegionEditFormPage = () => {
   };
 
   const handleSubmit = async (values: any) => {
-    const newRegion: any = {
-      regionName: values.regionName,
-      description: values.description,
-      phoneNumber: values.phoneNumber,
-      email: values.email,
-      link: values.link,
-      street: values.street,
-      houseNumber: values.houseNumber,
-      officeNumber: values.officeNumber,
-      postIndex: values.postIndex,
-      logo: logo,
-      city: values.city,
-      isActive: chosenRegion.isActive
-    };
-    await RegionsApi.EditRegion(currentRegion, newRegion);
+    setLoading(true);
 
-    form.resetFields();
+    const checkResponce = await checkIfNameExists(values.regionName);
 
-    notificationLogic("success", successfulEditAction("Дані округи"));
-    history.push(`/regions/${currentRegion}`);
+    if (checkResponce.data === false) {
```

Comment:
Try to do something like:
```suggestion
    if (await checkIfNameExists(values.regionName).data) {
```

Reply:
Done!